components/admin_console/schema_admin_settings.jsx

```diff
@@ -817,7 +817,7 @@ export default class SchemaAdminSettings extends React.Component {
         let config = JSON.parse(JSON.stringify(this.props.config));
         config = this.getConfigFromState(config);
 
-        await saveConfig(
+        await this.props.updateConfig(
             config,
             (savedConfig) => {
```

Comment:
Good job catching this @reflog

Since `updateConfig` returns a promise instead of using callbacks, passing in callbacks for the second and third parameters no longer works here. I came across this and made a commit here for using the promise `updateConfig` returns here:
https://github.com/mattermost/mattermost-webapp/commit/eabfb92b2e7058b322058d2807f5ad3eccb630d0#diff-532d22f84e6300c62626b313fab62fa0R820

Reply:
face-palm. :) surprisingly it did perform the save, but it's good that you've noticed this!